documentation/API_documentation/S2CVPN_User_Story.md

```diff
@@ -0,0 +1,10 @@
+# SiteToCloud(S2C)  VPN API User Story
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an application developer or industry customer, I want to request (using my application server/backend service)cloud leased network connection service between siteA/cloudA and  siteB/cloudB from a Communication Service Provider (CSP), so that I can ensure better and more reliable quality for our end users at any required time with proper cost. |
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User<br> **Actors:** Application service providers,  industry customer,  application developers.<br> **Scope:** Service Booking(check network availability level, resource availability and corresponding fees )   \- Create/Remove/Modify/Query  service instance (S2C VPN session resource) |
```

Comment:
For Scope here, do you think it's better to claify in usage of Site to Cloud VPN Service?

Reply:
Yes，agree